License2Deploy/rolling_deploy.py

```diff
@@ -222,11 +224,45 @@ def healthcheck_new_instances(self, group_name): # pragma: no cover
     self.wait_for_new_instances(new_instance_ids) #Wait for new instances to be up and ready
     self.lb_healthcheck(new_instance_ids) #Once instances are ready, healthcheck. If successful, decrease desired count.
 
+  def retrieve_all_cloudwatch_alarms(self):
+    """ Retrieve all the Cloud-Watch alarms for the given project and environment """
+    try:
+      cloud_watch_alarms = self.conn_cloudwatch.describe_alarms()
```

Comment:
Let's return the lambda, rather than setting it as a class variable: 
```
return filter(lambda alarm: self.project in alarm.name and self.env in alarm.name, cloud_watch_alarms)
```